extensions/binding/org.eclipse.smarthome.binding.bosesoundtouch/ESH-INF/thing/channels.xml

```diff
@@ -281,5 +281,10 @@
 			</options>
 		</state>
 	</channel-type>
-
+	<!-- Notification channels -->
+	<channel-type id="notificationsound">
```

Comment:
Could you use the identical definition [as in Sonos](https://github.com/eclipse/smarthome/blob/master/extensions/binding/org.eclipse.smarthome.binding.sonos/ESH-INF/thing/channels.xml#L134-L138)?